src/libsystemd-network/sd-dhcp-client.c

```diff
@@ -2181,6 +2190,28 @@ int sd_dhcp_client_stop(sd_dhcp_client *client) {
         return 0;
 }
 
+int sd_dhcp_client_initialize_timeout_ipv6_only_preferred(sd_dhcp_client *client, uint32_t delay) {
+        uint64_t usec = 0;
+        int r;
+
+        assert(client);
+        assert(client->event);
+
+        assert_se(sd_event_now(client->event, CLOCK_BOOTTIME, &usec) >= 0);
```

Comment:
Please use the timestamp of the lease (or maybe the timestamp of the sending request?).